internal/cli/run/run.go

```diff
@@ -74,6 +76,13 @@ func init() {
 
 	unattendedCmd := cmd.Command("unattended", "")
 	unattendedCmd.Action(func(_ *kingpin.ParseContext) error {
+		if runtime.GOOS == "darwin" {
+			// Until we have enabled the check-in API we're called every
+			// hour on darwin and we need to self throttle.
+			// TODO(bassosimone): switch to check-in and remove this hack.
+			const veryFew = 10
```

Comment:
@FedericoCeratto do you think this is a reasonable number? Should we bump it to something higher?

Reply:
He says ten is reasonable. We could also possibly use twenty. Because we will have check-in in 2-3 weeks, I think it's fine here to be conservative and just keep ten, which seems "low" to everyone. What do you think?

Reply:
Ok let's air on the safe side and keep it to 10. We can always make a new release. As we have seen homebrew users do update fairly regularly.